client/src/legacy/CMSMain.Tree.js

```diff
@@ -140,6 +141,46 @@ $.entwine('ss.tree', function($) {
         }
       };
       return config;
+    },
+
+    canMove: async function(data) {
+      // only display warning if its the homepage
+      var isHomePage = data.rslt.o.find(".homepage").first().length > 0;
+      if (!isHomePage) {
+        return true;
+      }
+
+      // only display warning if we're moving to a new parent
+      var oldParentId = data.rslt.op.data('id');
+      var newParentId = data.rslt.np.data('id');
+      if (oldParentId === newParentId) {
+        return true;
+      }
+
+      var message = i18n.sprintf(
+        i18n._t(
+          'CMS.RemoveHomePageWarningMessage',
+          'Warning: This page has the URL of %s. ' +
+          'This is required by your website to  identify the home page. ' +
+          'Without this, visitors will not be able to view your home page.'
+        ),
+        "<em>home</em>"
```

Comment:
Same comment as the previous file - as Guy suggested you can use `inject` if the injection is dynamic, but here it looks static, so you may as well just make it part of the base translation string